python/tvm/autotvm/graph_tuner/dynamic_programming_tuner.py

```diff
@@ -57,6 +59,8 @@ def __init__(self, *args, **kwargs):
             "input_shapes": self._input_shapes,
             "layout_transform_interlayer_cost": self._layout_transform_interlayer_cost
         }
+        self._opt_out_op = OPT_OUT_OP
```

Comment:
We don't need to do this again for sub class.

Reply:
I see. I will remove that.